File: app/src/main/java/com/taitsmith/busboy/BusboyApplication.java
package com.taitsmith.busboy;

import android.app.Application;

import net.danlew.android.joda.JodaTimeAndroid;

public class BusboyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        JodaTimeAndroid.init(this);
    }
}

File: app/src/main/java/com/taitsmith/busboy/BusboyApplication.kt
package com.taitsmith.busboy

import dagger.hilt.android.HiltAndroidApp
import android.app.Application
import com.google.android.gms.location.LocationServices
import dagger.Module
import dagger.Provides
import dagger.hilt.InstallIn
import dagger.hilt.android.components.ViewModelComponent
import dagger.hilt.components.SingletonComponent
import javax.inject.Singleton

@HiltAndroidApp
class BusboyApplication : Application()

@Module
@InstallIn(SingletonComponent::class)
object AppModule {

    @Provides
    @Singleton
    fun provideLocationProviderClient(application: Application) =
        LocationServices.getFusedLocationProviderClient(application)
}